spec/system/admin/orders_spec.rb

```diff
@@ -385,6 +385,9 @@
         end
 
         it "can bulk send invoice for 2 orders" do
+          Spree::Config[:enable_invoices?] = true
+          Spree::Config[:enterprise_number_required_on_invoices?] = false
```

Comment:
I assume this config was needed in order for the specs to pass?

Reply:
Yes!